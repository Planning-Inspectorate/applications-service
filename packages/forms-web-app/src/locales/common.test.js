const enCommonTranslations = require('./en/common.json');
const cyCommonTranslations = require('./cy/common.json');

describe('locales/common', () => {
	it('should return the correct English common translations', () => {
		expect(enCommonTranslations).toMatchSnapshot();
	});

	it('should return the correct Welsh common translations', () => {
		expect(cyCommonTranslations).toMatchSnapshot();
	});

	it('should return the same keys for English and Welsh translations', () => {
		expect(enCommonTranslations).toHaveSameKeys(cyCommonTranslations);
	});
});
