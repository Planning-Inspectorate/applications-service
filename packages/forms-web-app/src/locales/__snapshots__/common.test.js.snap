// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`locales/common should return the correct English common translations 1`] = `
{
  "acceptAndContinue": "Accept and continue",
  "applyFilters": "Apply filters",
  "change": "Change",
  "clearAllFilters": "Clear all filters",
  "clearFilters": "Clear filters",
  "clearSearch": "Clear search",
  "close": "Close",
  "contents": "Contents",
  "continue": "Continue",
  "dateFrom": "Date from",
  "datePublished": "Date published",
  "dateTo": "Date to",
  "day": "Day",
  "error": "Error",
  "filter": "Filter",
  "from": "From",
  "govUK": {
    "pressOfficeOpeningHours": "9:00 to 17:00 on Monday to Friday (except public holidays)",
  },
  "hide": "Hide",
  "hideAllSections": "Hide all sections",
  "hideAllSteps": "Hide all steps",
  "month": "Month",
  "next": "Next",
  "no": "No",
  "open": "Open",
  "partOf": "Part of",
  "pinsContactDetails": {
    "csOpeningHours": "Monday to Friday, 9am to 4pm (except bank holidays)",
  },
  "resultsPerPage": "Results per page",
  "returnToResults": "Return to results",
  "saveAndExit": "Save & Exit",
  "saveChanges": "Save changes",
  "search": "Search",
  "selectAllFilters": "Select all filters",
  "show": "Show",
  "showAllSections": "Show all sections",
  "showAllSteps": "Show all steps",
  "showFilters": "Show filters",
  "startNow": "Start now",
  "success": "Success",
  "telephone": "Telephone",
  "thereIsAProblem": "There is a problem",
  "to": "To",
  "validationErrors": {
    "aboutProject": {
      "empty": "Enter what you want to tell us about this proposed project",
      "length": "What you want to tell us must be {{-maxCharacters}} characters or less",
    },
    "address": {
      "countryEmpty": "Enter a country",
      "countryLength": "Country must be 64 characters or less",
      "line1Empty": "Enter address line 1",
      "line1Length": "Address line 1 must be 255 characters or less",
      "line2Length": "Address line 2 must be 96 characters or less",
      "line3Length": "Town or city must be 64 characters or less",
      "postcodeEmpty": "Enter a postcode",
      "postcodeLength": "Postcode must be 16 characters or less",
    },
    "areYou18": "Select yes if you are 18 or over",
    "emailAddress": {
      "empty": "Enter your email address",
      "format": "Enter an email address in the correct format, like name@example.com",
      "length": "Email address must be between 3 and 255 characters",
    },
    "fullName": {
      "empty": "Enter your full name",
      "length": "Full name must be between 3 and 64 characters",
    },
    "registeringFor": {
      "empty": "Select who you are registering for",
    },
    "telephone": {
      "empty": "Enter your telephone number",
      "format": "Enter a telephone number, like 01632 960 001, 07700 900 982 or 44 808 157 0192",
      "length": "Telephone number must be 255 characters or less",
    },
  },
  "year": "Year",
  "yes": "Yes",
}
`;

exports[`locales/common should return the correct Welsh common translations 1`] = `
{
  "acceptAndContinue": "Derbyn a pharhau",
  "applyFilters": "Cynnig hidlydd",
  "change": "Newid",
  "clearAllFilters": "Clirio pob hidlydd",
  "clearFilters": "Cliriwch hidlydd",
  "clearSearch": "Clirio chwiliad",
  "close": "Caeedig",
  "contents": "Cynnwys",
  "continue": "Parhau",
  "dateFrom": "Dyddiad o",
  "datePublished": "Dyddiad cyhoeddi",
  "dateTo": "Dyddiad i",
  "day": "Diwrnod",
  "error": "Gwall",
  "filter": "Hidlydd",
  "from": "Oddi wrth",
  "govUK": {
    "pressOfficeOpeningHours": "9:00 tan 17:00 o ddydd Llun i ddydd Gwener (heblaw am wyliau cyhoeddus)",
  },
  "hide": "Cuddio",
  "hideAllSections": "Cuddio pob adran",
  "hideAllSteps": "Cuddio pob cam",
  "month": "Mis",
  "next": "Nesaf",
  "no": "Nac ydw",
  "open": "Agor",
  "partOf": "Rhan o",
  "pinsContactDetails": {
    "csOpeningHours": "Dydd Llun i ddydd Gwener, 9am tan 4pm (heblaw am wyliau cyhoeddus)",
  },
  "resultsPerPage": "Canlyniadau fesul tudalen",
  "returnToResults": "Dychwelyd at ganlyniadau",
  "saveAndExit": "Cadw & Gadael",
  "saveChanges": "Achub newidiadau",
  "search": "Chwilio",
  "selectAllFilters": "Dewiswch bob hidlydd",
  "show": "Dangos",
  "showAllSections": "Dangos pob adran",
  "showAllSteps": "Dangos pob cam",
  "showFilters": "Dangos hidlydd",
  "startNow": "Dechrau nawr",
  "success": "Llwyddiant",
  "telephone": "Rhif ffôn",
  "thereIsAProblem": "Mae yna broblem",
  "to": "I",
  "validationErrors": {
    "aboutProject": {
      "empty": "Nodwch yr hyn rydych eisiau ei ddweud wrthym am y prosiect arfaethedig hwn",
      "length": "Mae'n rhaid i'r hyn rydych eisiau ei ddweud wrthym gynnwys {{-maxCharacters}} o nodau neu lai",
    },
    "address": {
      "countryEmpty": "Nodwch wlad",
      "countryLength": "Mae'n rhaid i'r wlad gynnwys 64 o nodau neu lai",
      "line1Empty": "Nodwch linell gyfeiriad 1",
      "line1Length": "Mae'n rhaid i linell gyfeiriad 1 gynnwys 255 o nodau neu lai",
      "line2Length": "Mae'n rhaid i linell gyfeiriad 2 gynnwys 96 o nodau neu lai",
      "line3Length": "Mae'n rhaid i'r dref neu'r ddinas gynnwys 64 o nodau neu lai",
      "postcodeEmpty": "Nodwch god post",
      "postcodeLength": "Mae'n rhaid i'r cod post gynnwys 16 o nodau neu lai",
    },
    "areYou18": "Dewiswch 'ydw' os ydych yn 18 oed neu'n hŷn",
    "emailAddress": {
      "empty": "Nodwch eich cyfeiriad e-bost",
      "format": "Nodwch gyfeiriad e-bost yn y fformat cywir, fel enw@enghraifft.com",
      "length": "Mae'n rhaid i'ch cyfeiriad e-bost gynnwys rhwng 3 a 255 o nodau",
    },
    "fullName": {
      "empty": "Nodwch eich enw llawn",
      "length": "Mae'n rhaid i'ch enw llawn gynnwys rhwng 3 a 64 o nodau",
    },
    "registeringFor": {
      "empty": "Dewiswch ar ran pwy rydych chi'n cofrestru",
    },
    "telephone": {
      "empty": "Nodwch eich rhif ffôn",
      "format": "Nodwch rif ffôn, fel 01632 960 001, 07700 900 982 neu 44 808 157 0192",
      "length": "Mae'n rhaid i'ch rhif ffôn gynnwys 255 o nodau neu lai",
    },
  },
  "year": "Blwyddyn",
  "yes": "Ydw",
}
`;
